tensorflow/core/kernels/eigen_pooling.h

```diff
@@ -305,6 +309,7 @@ struct AvgPoolMeanReducer {
 
 #if (EIGEN_ARCH_i386 || EIGEN_ARCH_x86_64) && !defined(__CUDACC__) && \
     !defined(__HIPCC__)
+
```

Comment:
extra blank line not needed.

Reply:
will remove the blank line in a different PR (as part of "cleanup trivial differences with upstream repo" PRs that I file periodically)